mm/ktsan/trace.c

```diff
@@ -66,6 +66,7 @@ void kt_trace_add_event(kt_thr_t *thr, kt_event_type_t type, uptr_t addr)
 		kt_trace_switch(trace, clock);
 
 	event.type = (int)type;
+	event.cpu_id = smp_processor_id();
```

Comment:
Yes, we can.

Should I rename pc to data and use it to store cpu id for thread start/stop events of should I add data bitfield like I did for cpu_id?